### ISSUE TITLE
okta-vue - replace path instead of push it

### ISSUE THREAD

**Issue by gaastonsr** [okta/okta-oidc-js#98]
When authentication is successful it redirects to a `callback` route in the app. And this route redirects to the page we were originally trying to access.

The problem is that the redirect is made with a push instead of a replace, that means that if the user hits the back button they will be back to the `callback` route which will fail.

```
Uncaught (in promise) n {name: "AuthSdkError", message: "Unable to parse a token from the url", errorCode: "INTERNAL", errorSummary: "Unable to parse a token from the url", errorLink: "INTERNAL", …}
```

Can we do the redirect with replace instead of push?

https://github.com/okta/okta-oidc-js/blob/f6f66aa3dfde922b4af5d80df655f07f7d37e5b0/packages/okta-vue/src/components/ImplicitCallback.vue#L6

**Comment by jmelberg-okta** [comment]
Great suggestion @gaastonsr! Want to submit a pull request and label it as a 🐛 ?

**Comment by jmelberg-okta** [comment]
Resolved with #160 and #167
